stratum/chassis_config.pb_with_PortShaping.txt

```diff
@@ -0,0 +1,79 @@
+channel: 1
+                                                                23,13         Top
```

Comment:
copy&paste error?

Reply:
Thanks for pointing this error. I already resolved.